### PR TITLE
Uyuni server and branch server install

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/16122
Esp. add missing info for Uyuni Retail.
[uyuni_retail_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/7520465/uyuni_retail_guide.pdf)
[suse_manager_retail_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/7520476/suse_manager_retail_guide.pdf)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I tried to address feedback provided by Julio.  Uyuni branch server description is still incomplete.  @aaannz can you please check what still missing or not correct?  In the initial description of the PR I linked the current PDF files.  I hope this is useful.

File: modules/retail/pages/retail-install-uyuni.adoc [hunk 1]
```diff
@@ -57,9 +57,18 @@ zypper in bind-formula \
...
++
+----
+yast2 susemanager_setup
+----
+. Follow the prompts to set up your account.
+  Take note of the passwords you set, you will need them later on.
+
+
+Continue with general {productname} configuration and channel synchronization.
+
```
Comment (aaannz) [inline modules/retail/pages/retail-install-uyuni.adoc:74]: I do not think this is necessary. I would leave the original text and instead fix the xref to point to https://github.com/uyuni-project/uyuni-docs/blob/master/modules/installation/pages/uyuni-server-setup.adoc

File: modules/retail/pages/retail-network-arch.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [[retail-network-arch]]
-= Network Architecture
+= Network Architecture and Branch Servers
```
Comment (aaannz) [inline modules/retail/pages/retail-network-arch.adoc:2]: Again, no need to add branch servers

File: modules/retail/pages/retail-uyuni-branchserver.adoc [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+== Trust GPG Keys on Clients
+
+include::snippets/trust_gpg.adoc[]
+
+
+
+// FIXME Starting from here, everything is preliminary
+// Feedback provided by Lukas
+== Create Activation Key for a Branch Server and the Retail Terminals
```
Comment (aaannz) [inline modules/retail/pages/retail-uyuni-branchserver.adoc:69]:
```suggestion
== Create Activation Key for a Branch Server and the Retail Terminals Images
```

File: modules/retail/pages/retail-uyuni-branchserver.adoc [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+The branch server is based on the {productname} Proxy.
+Its activation key must contain these child channels:
+
+* openSUSE Leap {opensuse-version} Updates (x86_64)
+* Uyuni Client Tools for openSUSE Leap {opensuse-version} (x86_64)
+* Uyuni Proxy Stable for openSUSE Leap {opensuse-version} (x86_64)
+
+The activation key for retail terminals based on openSUSE Leap {opensuse-version} must contain these child channels:
```
Comment (aaannz) [inline modules/retail/pages/retail-uyuni-branchserver.adoc:78]:
```suggestion
The activation key for retail terminals images based on openSUSE Leap {opensuse-version} must contain these child channels:
```

File: modules/retail/pages/retail-uyuni-branchserver.adoc [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+----
+retail_branch_init <branch_server_minion_id> --dedicated-nic eth1 \
+    --branch-ip 192.168.7.5 \
+    --netmask 255.255.255.0 \
+    --dyn-range 192.168.7.100 192.168.7.200 \
+    --server-domain branch.example.org \
+    --branch-prefix uyuni
+----
++
+  For additional options,  use the [command]``retail_branch_init --help`` command.
```
Comment (aaannz) [inline modules/retail/pages/retail-uyuni-branchserver.adoc:146]: This is run on Uyuni Server. So start with something "On Uyuni server run `retail_branch_init ...`"

**Review by aaannz — CHANGES_REQUESTED** [review]

**Review by aaannz — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/retail/pages/retail-install-uyuni.adoc (hunk 1 above)
Comment (keichwa) [reply modules/retail/pages/retail-install-uyuni.adoc]: ok, reverted now and xref provided.

**Pull request merged**